### PR TITLE
Update k0s version to 1.31.1+k0s.1

### LINKED ISSUE #523: Update k0s version to 1.31.1+k0s.1

### PR DESCRIPTION
**Pull request by Kshatrix**
Resolves #523

### CODE REVIEW

File: templates/provider/hmc-templates/files/templates/aws-standalone-cp-0-0-1.yaml
Comment (eromanova) [inline templates/provider/hmc-templates/files/templates/aws-standalone-cp-0-0-1.yaml:1]: Should the new template files be regenerated?

**Review by eromanova — CHANGES_REQUESTED** [review]

File: templates/provider/hmc-templates/files/templates/aws-standalone-cp-0-0-1.yaml
Comment (Kshatrix) [reply templates/provider/hmc-templates/files/templates/aws-standalone-cp-0-0-1.yaml:1]: yep, left them uncommited, thanks :D

File: templates/cluster/azure-standalone-cp/Chart.yaml
Comment (eromanova) [inline templates/cluster/azure-standalone-cp/Chart.yaml:1]: Version should also be updated to `0.0.2`

**Review by eromanova — CHANGES_REQUESTED** [review]
Let's update `spec.template` in `config/dev/*-managedcluster.yaml` file. Also, could you please rerun `templates-generate` to generate azure-standalone template?

**Review by eromanova — APPROVED** [review]

**Pull request merged**